### DISCUSSION TITLE
Manually select other zigbee device for communication

### DISCUSSION THREAD

**Discussion by pmol200** [Koenkk/zigbee2mqtt#10836] [Devices]
I am clicking around in the zigbee2mqtt interface and see some settings that are interesting. I have various sensors (door, smoke detection, temperature) which are spread out over 3 floors. 

one of my devices is far away from the coordinator, which is two floors lower than the sensor. The link quality (LQI ?) is 0, according to the zigbee2mqtt interface.

When opening the sensor properties, there is an option to change bindings. I thought this would be something automatically, devices find each other when they are not to far away and create a chain. but now I am starting to think I need to configure this manually.

Is that correct ? should I manually define a sensor that is near to the one that has a poor link quality? Is that something that should be done for all sensors ? just define near sensors so they have a better communication?

**Comment by sjorge** [comment]
No, binding is something different.

It's to bind functionality of 2 devices together, for example bind a remote to a bulb so it can turn the bulb off and on.

**Reply by pmol200** [reply]
Ok, thanks for the clarification.
So I don't need to tweak anything to get optimal connectivity of all sensors.

If I am right the LQI value says something about the signal strength, as very close by sensors have a high value and sensors further away have a lower value.

What looks strange to me is that I have smoke sensors on 3 levels of my house and the 2nd level the LQI value is ok and the level above the LQI is zero. there is about 3 meters between the two sensors and it is above the stairs, so no wall or floor in between. Why isn't the signal from the 2nd level forwarded to the sensor on the 3rd level ? 

Looking at the LQI values, I can only conclude that the signal goed directly from the USB concentrator to the top sensor and not amplified by the other sensor(s).

anyone know why this happens?

I tought zigbee is smart and all sensors amplify the signal to reach out to other sensors.
